src/Controls/tests/DeviceTests/ControlsDeviceTestExtensions.cs

```diff
@@ -13,6 +14,31 @@ namespace Microsoft.Maui.DeviceTests
 {
 	public static class ControlsDeviceTestExtensions
 	{
+
+		public static bool IsAccessibilityElement(this VisualElement view)
+		{
+			bool result = AssertionExtensions.IsAccessibilityElement(view);
+
+#if IOS
+			// UIView.IsAccessibilityElement will never be true if VoiceOver isn't running
+			//
+			// Because the 'AssertionExtensions.IsAccessibilityElement' just fakes the actual values
+			// when VO isn't running we can't really test that calling `SetIsInAccessibleTree` with
+			// false works unless we turn on VO
+			if (!UIKit.UIAccessibility.IsVoiceOverRunning)
+			{
+				bool? expected = AutomationProperties.GetIsInAccessibleTree(view);
+
+				if (expected.HasValue && !expected.Value)
```

Comment:
I'm a bit lost on this if statement. Could you explain the `!expected.Value` part to me?

Reply:
Deleted this extension method and reworked the non `VoiceOver` path